tests/test_quantity.py

```diff
@@ -0,0 +1,437 @@
+    assert convert.unit == "in^2"
+
+
+def test_to_26():
+    v = q.Quantity(1.0, "radian s^-1")
+    convert = v.to("degree s^-1")
+    assert convert.value == pytest.approx(57.295779, DELTA)
+    assert convert.unit == "degree s^-1"
+
+
+def test_to_27():
+    v = q.Quantity(1.0, "degree s^-1")
+    convert = v.to("radian s^-1")
+    assert convert.value == pytest.approx(0.01745329251, DELTA)
+    assert convert.unit == "radian s^-1"
+
+
+def test_to_28():
+    v = q.Quantity(1.0, "dyne cm^-2")
+    convert = v.to("N m^-2")
+    assert convert.value == pytest.approx(0.1, DELTA)
+    assert convert.unit == "N m^-2"
+
+
+def test_to_29():
+    v = q.Quantity(1.0, "psi")
+    convert = v.to("Pa")
+    assert convert.value == pytest.approx(6894.76, DELTA)
+    assert convert.unit == "Pa"
+
+
+def test_to_30():
+    v = q.Quantity(1.0, "pdl")
+    convert = v.to("N")
+    assert convert.value == pytest.approx(0.138254999, DELTA)
+    assert convert.unit == "N"
+
+
+def test_to_31():
+    v = q.Quantity(1.0, "ohm cm")
+    convert = v.to("ohm m")
+    assert convert.value == pytest.approx(0.01, DELTA)
+    assert convert.unit == "ohm m"
+
+
+def test_to_32():
+    v = q.Quantity(1.0, "erg")
+    convert = v.to("J")
+    assert convert.value == pytest.approx(1.0e-7, DELTA)
+    assert convert.unit == "J"
+
+
+def test_to_33():
+    v = q.Quantity(1.0, "BTU")
+    convert = v.to("J")
+    assert convert.value == pytest.approx(1055.056, DELTA)
+    assert convert.unit == "J"
+
+
+def test_to_34():
+    v = q.Quantity(1.0, "gal")
+    convert = v.to("m^3")
+    assert convert.value == pytest.approx(0.00378541, DELTA)
+    assert convert.unit == "m^3"
+
+
+def test_to_35():
+    v = q.Quantity(1.0, "l")
+    convert = v.to("m^3")
+    assert convert.value == pytest.approx(0.001, DELTA)
+    assert convert.unit == "m^3"
+
+
+def test_to_36():
+    v = q.Quantity(1.0, "BTU lb^-1 R^-1")
+    convert = v.to("J kg^-1 K^-1")
+    assert convert.value == pytest.approx(4186.8161854, DELTA)
+    assert convert.unit == "J kg^-1 K^-1"
+
+
+def test_to_37():
+    v = q.Quantity(1.0, "BTU lb^-1 F^-1")
+    convert = v.to("J kg^-1 K^-1")
+    assert convert.value == pytest.approx(4186.8161854, DELTA)
+    assert convert.unit == "J kg^-1 K^-1"
+
+
+def test_to_38():
+    v = q.Quantity(1.0, "gal^-1")
+    convert = v.to("m^-3")
+    assert convert.value == pytest.approx(264.172, DELTA)
+    assert convert.unit == "m^-3"
+
+
+def test_to_39():
+    v = q.Quantity(1.0, "BTU ft^-2")
+    convert = v.to("J m^-2")
+    assert convert.value == pytest.approx(11356.5713242, DELTA)
+    assert convert.unit == "J m^-2"
+
+
+def testing_dimensions():
+    print(f"{'*' * 25} {testing_dimensions.__name__} {'*' * 25}")
+
+    def dim_test(unit_str, dim_list):
+        qt = q.Quantity(10, unit_str)
+        print(f"{unit_str} : {qt.get_dimensions_list()}")
+        assert qt.get_dimensions_list() == dim_list
+
+    dim_test("m", [0, 1, 0, 0, 0, 0, 0, 0, 0, 0])
```

Comment:
You should add tests for things like `s^0.5`.

Reply:
Done. Thank you.